.github/workflows/mirror.yml

```diff
@@ -0,0 +1,24 @@
+name: Mirror
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  release:
+    types: [published]
+
+  
+jobs:
+  mirror:
+    name: Mirror images
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+
+    steps:
+      - name: Mirror docker images to Github Registry
+        run: |
+          skopeo sync --src docker --dest docker docker.io/kindest/node ghcr.io/kind-ci
```

Comment:
End users will want this mirrored image as well :+1:, so maybe `-ci` suffix can be omitted.